pyspectral/blackbody.py

```diff
@@ -99,24 +99,17 @@ def planck(wave, temperature, wavelength=True):
         LOG.debug("Using {0} when calculating the Blackbody radiance".format(
             units[(wavelength is True) - 1]))
 
-    if np.isscalar(temperature):
-        temperature = np.array([temperature, ], dtype='float64')
-    elif isinstance(temperature, (list, tuple)):
-        temperature = np.array(temperature, dtype='float64')
-
+    temperature = _scalar_tuple_list_to_numpy(temperature)
     shape = temperature.shape
-    if np.isscalar(wave):
-        wln = np.array([wave, ], dtype='float64')
-    else:
-        wln = np.array(wave, dtype='float64')
+    wave = _scalar_tuple_list_to_numpy(wave)
 
     if wavelength:
         const = 2 * H_PLANCK * C_SPEED ** 2
-        nom = const / wln ** 5
-        arg1 = H_PLANCK * C_SPEED / (K_BOLTZMANN * wln)
+        nom = const / wave ** 5
+        arg1 = H_PLANCK * C_SPEED / (K_BOLTZMANN * wave)
```

Comment:
In #203, we introduced these constants
https://github.com/pytroll/pyspectral/blob/main/pyspectral/blackbody.py#L39-L40
Could they be used here?

Reply:
Done in https://github.com/pytroll/pyspectral/pull/206/commits/0c2a51f20fdb0f8b5e0fba2de1898df72cbf15f3